pcl_conversions/include/pcl_conversions/pcl_conversions.h

```diff
@@ -77,27 +79,28 @@ namespace pcl_conversions {
   /** PCLHeader <=> Header **/
 
   inline
-  void fromPCL(const pcl::uint64_t &pcl_stamp, ros::Time &stamp)
+  void fromPCL(const pcl::uint64_t &pcl_stamp, rclcpp::Time &stamp)
   {
-    stamp.fromNSec(pcl_stamp * 1000ull);  // Convert from us to ns
+    stamp = rclcpp::Time(pcl_stamp * 1000ull); // Convert from us to ns
+    //stamp.nanoseconds(pcl_stamp * 1000ull);
```

Comment:
Why not use the commended out version? Also if that doesn't work, then remove from being commented out and delete it